src/Resources/config/services.yml

```diff
@@ -92,6 +92,17 @@ services:
         class: Contao\CoreBundle\Security\User\ContaoUserProvider
         public: false
 
+    contao.store.session:
```

Comment:
Yes, I know :-D But what I was saying is that as this is till WIP it will be renamed completely if we decide that it should also be responsible to load the values again on kernel request :)